ui/src/Application/Styleguide/Main.scss

```diff
@@ -61,6 +63,12 @@ a.deleteLink {
 
 input[type="text"] {
   padding-left: 8px;
+
+  &:focus {
+    outline: none;
+    border: $focus-state-border !important;
```

Comment:
Using "important" is a little smelly to me usually, but I'm curious to hear why it's needed

Reply:
I agree, don't like this, but it wasn't really feasible to make the selector specific enough here.